### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by peterjah**

### CODE REVIEW

**Review by pivilartisant — APPROVED** [review]

**Review by Pierregvx — APPROVED** [review]

File: web-frontend/src/pages/CreateAccount/StepThree.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default function StepThree() {
   );
 
   if (isSuccess) {
-    navigate(routeFor('homepage'), { state: { nickname } });
+    navigate(routeFor('dashboard'), { state: { nickname } });
```
Comment (Thykof) [inline web-frontend/src/pages/CreateAccount/StepThree.tsx:25]: there is no /dashboard route

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx
Comment (Thykof) [inline web-frontend/src/pages/Home/Wallet/Wallet.tsx:1]: if it's a layout, why not putting this file into layouts folder and name it WalletLayout?

File: web-frontend/src/pages/SelectAccount/SelectAccount.tsx [hunk 2]
```diff
@@ -37,14 +37,19 @@ export default function SelectAccount() {
...
-              <div className="mb-4" key={account.nickname}>
-                <AccountSelector
-                  avatar={<FiUser className="text-neutral h-6 w-6" />}
-                  icon={<MassaToken size={24} />}
-                  accountName={account.nickname}
-                  amount={toMAS(account.candidateBalance).toString()}
-                />
-              </div>
+              <Link
+                to={routeFor('home')}
```
Comment (Thykof) [inline web-frontend/src/pages/SelectAccount/SelectAccount.tsx:41]: is there a warning like `Each child in an array should have a unique "key" prop.`?

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  const { data: accounts = [] } = useResource<AccountObject[]>('accounts');
+
+  // If no account, redirect to welcome page
+  if (!accounts.length) {
+    navigate(routeFor('index'));
+  }
+
+  // const [selectedAccount, setSelectedAccount] = useState<AccountObject>(
+  //   accounts.find((account) => account.nickname === nickname)!,
+  // );
```
Comment (Thykof) [inline web-frontend/src/pages/Home/Wallet/Wallet.tsx:47]: why is it commented out?

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  const [selectedItem, setSelectedItem] = useState<MenuItem>(
+    state.menuItem ?? MenuItem.SendReceive,
+  );
+
+  const isActive = (item: MenuItem) => {
+    return item === selectedItem;
+  };
+
+  let menuConf = {
+    title: 'MassaWallet',
```
Comment (Thykof) [inline web-frontend/src/pages/Home/Wallet/Wallet.tsx:58]: put this into i18n

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    logo: <MassaWallet />,
+    fullMode: true,
+  };
+
+  let menuItems = [
+    {
+      label: 'Wallet',
```
Comment (Thykof) [inline web-frontend/src/pages/Home/Wallet/Wallet.tsx:65]: put all the label into i18n.

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+      active: isActive(MenuItem.LightTheme),
+      footer: true,
+      onClickItem: () => setSelectedItem(MenuItem.LightTheme),
+    },
+  ];
+
+  const accountsItems = accounts.map((account) => ({
+    icon: <FiUser size={32} />,
+    item: account.nickname,
+    // onClick: () => setSelectedAccount(account),
```
Comment (Thykof) [inline web-frontend/src/pages/Home/Wallet/Wallet.tsx:118]: why is it commented out? it's rarely a good practice to add commented out code, at least add a comment to explain why you commented it.

**Review by Thykof — CHANGES_REQUESTED** [review]

File: web-frontend/src/pages/CreateAccount/StepThree.tsx (hunk 1 above)
Comment (pivilartisant) [reply web-frontend/src/pages/CreateAccount/StepThree.tsx]: changed to `home`

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx (hunk 3 above)
Comment (pivilartisant) [reply web-frontend/src/pages/Home/Wallet/Wallet.tsx:47]: legacy deleted in changes

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx (hunk 6 above)
Comment (pivilartisant) [reply web-frontend/src/pages/Home/Wallet/Wallet.tsx:118]: legacy deleted in changes

File: web-frontend/src/pages/Home/Wallet/Wallet.tsx (hunk 4 above)
Comment (pivilartisant) [reply web-frontend/src/pages/Home/Wallet/Wallet.tsx]: is it best to create a seperate issue ?

File: web-frontend/src/layouts/WalletLayout/WalletLayout.tsx [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+  const accountsItems = accounts.map((account) => ({
+    icon: <FiUser size={32} />,
+    item: account.nickname,
+  }));
+
+  const selectedAccountKey: number = parseInt(
+    Object.keys(accounts).find(
+      (_, idx) => accounts[idx].nickname === nickname,
+    )!,
```
Comment (Thykof) [inline web-frontend/src/layouts/WalletLayout/WalletLayout.tsx:119]: linter complain about the `!` (non null assertion).

**Review by Thykof — APPROVED** [review]

**Pull request merged**